angrmanagement/ui/workspace.py

```diff
@@ -143,6 +144,34 @@ def on_function_tagged(self):
     # Public methods
     #
 
+    def new_disassembly_view(self):
+        """
+        Add a new disassembly view into the
+        central_widget and rename the views
+        accordingly
+
+        :return:    None
+        """
+
+        if self.view_manager.disas_view_counter == 1:
+            for view in self.view_manager.docks:
+                if view.windowTitle() == 'Disassembly':
+                    view.setWindowTitle("Disassembly-A")
+                    self.view_manager.first_view_in_category('disassembly').caption = "Disassembly-A"
+                    break
+        if self.view_manager.disas_view_counter < 26:
```

Comment:
Perhaps comments might be nice right here, as just from a glance it's unclear how you got `26` as view count number.

Reply:
I'll make these changes once I am back in town on Monday!